src/System.Memory/ref/System.Memory.cs

```diff
@@ -494,12 +494,18 @@ public static partial class Utf8Parser
 }
 namespace System.Runtime.InteropServices
 {
+    using System.Buffers;
```

Comment:
We don't do this in contracts.  When this file is auto-generated by the tool at some point again, this will be removed and full names will be used everywhere.  Might as well do so now.

Reply:
Was having issues getting it compile; didn't know needed a change to coreclr also/first https://github.com/dotnet/coreclr/pull/16455